Documentation/guides.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<guides xmlns="https://www.phpdoc.org/guides"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+        links-are-relative="true"
+>
+    <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+               project-home="https://docs.typo3.org/m/typo3/tutorial-typoscript-in-45-minutes/main/en-us/"
+               project-contact="https://typo3.slack.com/archives/C028JEPJL"
+               project-repository="https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Typoscript45Minutes"
+               project-issues="https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Typoscript45Minutes/issues"
+               edit-on-github-branch="main"
+               edit-on-github="TYPO3-Documentation/TYPO3CMS-Tutorial-Typoscript45Minutes"
+               typo3-core-preferred="stable"
```

Comment:
```suggestion
               typo3-core-preferred="main"
```
Change this to main, then most interlink inventories bellow should be unnessesary